tests/integration/conftest.py

```diff
@@ -65,6 +65,11 @@ def app():
         TESTING=True,
     )
 
+    # Switch to the test db, if configured (in the local config file).
+    db_uri_test = app.config.get('SQLALCHEMY_DATABASE_URI_TEST')
```

Comment:
This feels strange, why would you have to configure two different db urls? Wouldn't you just change the actual `SQLALCHEMY_DATABASE_URI` locally and that's it?